js/libs/keycloak-js/lib/keycloak.js

```diff
@@ -673,7 +673,7 @@ function Keycloak (config) {
 
                                 kc.onAuthRefreshError && kc.onAuthRefreshError();
                                 for (var p = refreshQueue.pop(); p != null; p = refreshQueue.pop()) {
-                                    p.setError(true);
+                                    p.setError("Failed to refresh the token. Status: " + req.status);
```

Comment:
Let's not add any status codes or network details here, that is what the network inspector for the devtools are for.
```suggestion
                                    p.setError("Failed to refresh token: An unexpected HTTP error occurred while attempting to refresh the token.");
```